src/org/usfirst/frc4904/robot/subsystems/LIDAR.java

```diff
@@ -1,20 +1,52 @@
 package org.usfirst.frc4904.robot.subsystems;
 
 
-import org.usfirst.frc4904.standard.custom.motioncontrollers.MotionController;
+import org.usfirst.frc4904.robot.commands.LIDARTurnbaugh;
+import org.usfirst.frc4904.standard.custom.motioncontrollers.CustomPIDController;
+import org.usfirst.frc4904.standard.custom.sensors.CANEncoder;
+import org.usfirst.frc4904.standard.custom.sensors.CANSensor;
 import org.usfirst.frc4904.standard.subsystems.motor.VelocitySensorMotor;
+import edu.wpi.first.wpilibj.PIDSourceType;
 import edu.wpi.first.wpilibj.SpeedController;
+import edu.wpi.first.wpilibj.command.Subsystem;
 
-public class LIDAR extends VelocitySensorMotor {
```

Comment:
It consists of several subcomponents, I did not think it made sense to prefer one of the components.